sls-server/serverless.yml

```diff
@@ -7,11 +7,12 @@ custom:
   premiseTableName: 'java-premise-${self:provider.stage}'
   incidentTableName: 'java-incident-${self:provider.stage}'
   noteTableName: 'java-note-${self:provider.stage}'
+  s3BucketName: 's3-bucket-${self:provider.stage}'
 
 provider:
   name: aws
   runtime: java8
-  stage: ${opt:stage, 'dev'}
+  stage: ${opt:stage, 'uat'}
```

Comment:
let keep default value as 'dev', we will do uat manually, so that we can control the deployment to uat.